scripts/k8s-versions-test.sh

```diff
@@ -22,36 +22,12 @@ function add_hosts {
 function add_helm_repos { 
   ## add the helm repos required to install and run ML and the v14 PoC
         printf "==> add the helm repos required to install and run ML and the v14 PoC\n" 
-        su - vagrant -c "helm repo add mojaloop http://mojaloop.io/helm/repo/ > /dev/null 2>&1 "
-        su - vagrant -c "helm repo add kiwigrid https://kiwigrid.github.io  > /dev/null 2>&1 "
-        su - vagrant -c "helm repo add elastic https://helm.elastic.co  > /dev/null 2>&1 "
-        su - vagrant -c "helm repo add bitnami https://charts.bitnami.com/bitnami > /dev/null 2>&1 "
-        su - vagrant -c "helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx  > /dev/null 2>&1 "
-        su - vagrant -c "helm repo update > /dev/null 2>&1 "
-}
-
-function update_charts {
-  ## update the PoC charts and dependencies
-        declare -a charts=(
-                dependencies/backend
-                mojaloop/common
-                mojaloop/admin-api-svc
-                mojaloop/fspiop-transfer-api-svc
-                mojaloop/mojaloop
-        )
-
-        printf "==> Updating helm Charts...\n"
-        for chart in "${charts[@]}"
-        do
-                printf  "    Updating chart: $chart    "
-                rm -rf $BASE_DIR/charts/$chart/charts 
-                if [[ `helm dep up $BASE_DIR/charts/$chart --skip-refresh 2>&1 | grep "Error" ` != "" ]] ; then 
-                        printf "[failed]\n"
-                        printf "Error updating chart:  %s \n" $chart 
-                else 
-                        printf "[ok]\n" $?
-                fi
-        done
+        su - $k8s_user -c "helm repo add mojaloop http://mojaloop.io/helm/repo/ > /dev/null 2>&1 "
```

Comment:
I dont think any of these are necessary except for the ingress-nginx.

All other dependencies will be pulled by our helm charts.

Reply:
Ok, I'll take them out